apps/test/unit/redux/unitSelectionReduxTest.js

```diff
@@ -86,60 +87,106 @@ describe('unitSelectionRedux', () => {
         expect(getSelectedScriptDescription(state)).toEqual(null);
       });
     });
+  });
```

Comment:
leave `describe('getSelectedScriptDescription'` outside of `describe('getSelectedUnitName'`?